essential-tests/src/testFixtures/kotlin/org/stellar/anchor/platform/e2etest/Sep24End2EndTests.kt

```diff
@@ -113,13 +122,53 @@ open class Sep24End2EndTests : AbstractIntegrationTests(TestConfig()) {
     assertCallbacks(actualCallbacks, getExpectedDepositStatus())
   }
 
+  @Test
+  fun `test deposit to contract address end-to-end flow`() = runBlocking {
+    val webAuthDomain = toml.getString("WEB_AUTH_FOR_CONTRACTS_ENDPOINT")
+    val homeDomain = "http://${URI.create(webAuthDomain).authority}"
+
+    val sep45Client =
+      Sep45Client(toml.getString("WEB_AUTH_FOR_CONTRACTS_ENDPOINT"), rpc, CLIENT_WALLET_SECRET)
+    val challenge =
+      sep45Client.getChallenge(
+        ChallengeRequest.builder()
+          .account(CLIENT_SMART_WALLET_ACCOUNT)
+          .homeDomain(homeDomain)
+          .build()
+      )
+    val token = sep45Client.validate(sep45Client.sign(challenge)).token
+    val sep24Client = Sep24Client(toml.getString("TRANSFER_SERVER_SEP0024"), token)
+
+    val request =
+      mapOf(
+        "asset_code" to "USDC",
+        "asset_issuer" to "GDQOE23CFSUMSVQK4Y5JHPPYK73VYCNHZHA7ENKCV37P6SUEO6XQBKPP",
+        "amount" to "1",
+      )
+    val response = sep24Client.deposit(request)
+
+    // Assert the interactive URL JWT is valid
+    val params = UriComponentsBuilder.fromUriString(response.url).build().queryParams
+    val cipher = params["token"]!![0]
+    val interactiveJwt = jwtService.decode(cipher, Sep24InteractiveUrlJwt::class.java)
+    assertEquals(CLIENT_SMART_WALLET_ACCOUNT, interactiveJwt.sub)
```

Comment:
Can you add the `CLIENT_SMAT_WALLET_ACCOUNT` created in the [testnet reset script](https://github.com/stellar/useful-scripts/blob/main/anchor-platform-testnet-reset/src/main/kotlin/Main.kt)?


```CLIENT_SMART_WALLET_ACCOUNT = "CBP7UV5XSPZ3NB2RDW7WSMZVFXD5XNRFNMOBG5DVI23BGCYIFOQHWE66"
```